test/test_angle.cpp

```diff
@@ -40,35 +42,38 @@ TEST(TestAngle, AngleArithmeticOperations)
   const auto angle_pi_6{cp::Angle{0.523598776_rad}};
```

Comment:
I see these pi values are being manually set, should they also be boost pi values for consistency or is this just for testing comparison

Reply:
It's only an example value used for testing.